File: C++/Daily Flash/week1/Day2_Jan14/code_c++/program2.cpp
/*
Program 2: Write a Program to check whether the Character is Vowel or
Consonant.
Input: E
Output: E is a Vowel.

*/
#include <iostream>

int main() {
	char ch;
	std::cout << "Enter character: ";
	std::cin >> ch;
	
	if (ch == 'a' || ch == 'i' || ch=='e' || ch== 'o' || ch == 'u' || ch == 'A' ||ch == 'E' || ch == 'I' || ch == 'O'||ch=='U' ) 
		std::cout << ch <<" is a vowel" << std::endl;
	else
		std::cout << ch <<" is consonant" << std::endl;
	return 0;	
}

File: PYTHON/Daily Flash/week1/Day2_Jan14/code_python/program2.py
'''
Program 2: Write a Program to check whether the Character is Vowel or
Consonant.
Input: E
Output: E is a Vowel.
'''


char = input("Enter character: ")

if(char == 'a' or char == 'e' or char == 'i' or char == 'o' or char == 'u'):
    print(char, " is a Vowel")
elif (char == 'A' or char == 'E' or char == 'I' or char == 'O' or char == 'U'):
    print(char, " is a Vowel")
else:
    print(char, " is a Consonant")
